templates/tg-mini-app-boilerplate-template/README.md

```diff
@@ -0,0 +1,45 @@
+## Overview
+
+This template is made from the [boilerplate template](https://github.com/aptos-labs/create-aptos-dapp/tree/main/templates/boilerplate-template), the difference is that we make this possible to run as a Telegram mini app, and we change the app to be a simple clicker game.
+
+The difference between this template and the [tg-mini-app-seamless-signing-boilerplate-template](https://aptos.dev/en/build/create-aptos-dapp/templates/tg-mini-app-seamless-signing-boilerplate) is we are using Aptos wallet adapter instead of Mizu wallet sdk core, this way the UX is similar to all other dapps using the Wallet Adapter, i.e. users need to approve the transaction explicitly. Also since Mizu is the only Aptos wallet supports Telegram now, we removed other wallet options.
+
+You can access a live demo of the mini app by starting a chat with the example bot https://t.me/aptos_explict_siging_bot/.
+
+To deploy your own mini app, you need to create a bot, deploy the dapp by following the [deployment guide](https://aptos.dev/en/build/create-aptos-dapp/templates/boilerplate#deploy-to-a-live-server), link the web app to the bot. You can follow a complete guide on the [mini app doc](https://docs.telegram-mini-apps.com/platform/creating-new-app).
+
+To debug and test the mini app, follow the guide on [mini app doc](https://docs.telegram-mini-apps.com/platform/debugging).
+
+## Create Aptos Dapp Boilerplate Template
```

Comment:
Lets have this whole README section mentions and talks about this template, since this is not a boilerplate template but a clicker game telegram template